changelog/std-format-fixDigitGrouping

```diff
@@ -0,0 +1,16 @@
+
+Fixed digits grouping for floating point number without decimal part
+
+$(REF formatValue, std, format) now correctly groups digits in the output string
+by inserting a group separator character (,) every n characters specified
+in the $(LINK2 https://dlang.org/library/std/format/formatted_write.html#format-string, Separator)
+grammar rule even if floating point numbers do not contain any decimal part.
+
+-------
+import std.format;
+
+assert(format("%,3.2f", 1172.009) == "1,172.01");
```

Comment:
It might be a good idea to mention the previous behavior as well so that people can easily judge whether this affects them.

Reply:
done